CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ### Added
 - ruby 2.4 testing in travis (@majormoses)
 
+### Added
```

Comment:
You can get rid of this and just leave the line below under the existing `### Added` tag.